metabrainz/views.py

```diff
@@ -1,4 +1,4 @@
-from flask import Blueprint, render_template
+from flask import Blueprint, render_template ,redirect , url_for
```

Comment:
The spaces are all over the place here. It should be `foo, bar` – no space before the comma, one space after it, just as in English text.